docs/ja/tutorial/05-functions.md

```
@@ -87,4 +87,182 @@ greet()   # 42
+
+```python
+function connect(host: str, port: int = 8080, timeout: int = 30) -> str:
+    return f"{host}:{port} (timeout={timeout})"
+
+print(connect("localhost"))              # localhost:8080 (timeout=30)
+print(connect("localhost", 3000))        # localhost:3000 (timeout=30)
+print(connect("localhost", 3000, 10))    # localhost:3000 (timeout=10)
+```
+
+> **なぜデフォルト引数なのか?** 一般的なケースではシンプルな呼び出しを保ちつつ、必要に応じてカスタマイズできます -- 複数のオーバーロードが不要になります。
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)